ui/src/app/exam/editor/publication/examPublication.component.html

```diff
@@ -100,17 +100,28 @@
         </button>
     </div>
     <div class="col-md-6 justify-content-center">
-        <ul class="list-inline">
-            <li *ngFor="let config of exam.examinationEventConfigurations" class="list-inline-item">
-                <span (click)="modifyExaminationEvent(config)" class="pointer">{{
-                    config.examinationEvent.start | date
-                }}</span>
+        <ul class="list-inline padt20">
+            <li
+                *ngFor="let config of sortByString(exam.examinationEventConfigurations)"
```

Comment:
Tässä vois varmaan käyttää suoraan orderBy-pipeä:

`*ngFor="let config of exam.examinationEventConfigurations | orderBy 'examinationEvent.start'"`

https://github.com/CSCfi/exam/blob/master/ui/src/app/utility/sorting/orderBy.pipe.ts

Reply:
Kokeilin sitä mut en saanu sitä noin toimimaan. Ei mm. buildaa.

Reply:
Jaa, no antaapa olla noin sitten.